packages/discord-bot/src/utils/getUserAccount.ts

```diff
@@ -1,27 +1,82 @@
-import { UserAccountModel } from 'api/dist/useraccount/entities';
-import { UserAccount, UserAccountDocument } from 'api/dist/useraccount/types';
-import { GuildMember } from 'discord.js';
+import { User } from 'discord.js';
+import { UserAccount } from './api-schema';
+import { apiClient } from './api';
+
+const createUserAccount = async (
+  user: User,
+  guildId: string
+): Promise<UserAccount[]> => {
+  await apiClient.post(
+    `/useraccounts`,
+    {
+      accountId: user.id,
+      name: user.username + '#' + user.discriminator,
+      avatarId: user.avatar || '',
+      platform: 'DISCORD',
+    },
+    {
+      headers: { 'x-discord-guild-id': guildId },
+    }
+  );
+  const response = await apiClient.get<UserAccount[]>(
+    `/useraccounts/?accountId=${user.id}`,
+    {
+      headers: { 'x-discord-guild-id': guildId },
```

Comment:
Header to send to api to connect call with the right community is `host`. 

See here for details: https://www.notion.so/giveth/Give-praise-using-Discord-Bot-2dc3710f4b644d47b9a1d6caba00cceb?pvs=4